lib/subscriber_list_mover.rb

```diff
@@ -48,4 +53,30 @@ def call
 
     puts "#{subscribers.count} active subscribers moved from #{from_slug} to #{to_slug}"
   end
+
+  def email_change_to_subscribers(source_subscriber_list)
+    email_subject = "Changes to GOV.UK email alerts"
+    email_utm_parameters = {
+        utm_source: email_subject,
+        utm_medium: "email",
+        utm_campaign: "govuk-subscription-ended",
+    }
+    email_redirect = EmailTemplateContext.new.add_utm("https://gov.uk/email/manage", email_utm_parameters)
+    list_title = source_subscriber_list.title.freeze
+
+    bulk_move_template = <<~BODY.freeze
+      Hello,
+      You've subscribed to get email alerts about #{list_title} information.
```

Comment:
Minor: I'm guessing this will come up in a content review, but we probably want some new lines here.

Reply:
Copy paste error